runtime/jvmti/jvmtiCapability.c

```diff
@@ -478,8 +478,10 @@ jvmtiRelinquishCapabilities(jvmtiEnv* env,
 
 #if JAVA_SPEC_VERSION >= 11
 		if (capabilities_ptr->can_generate_sampled_object_alloc_events) {
-			/* The default sampling interval is not changed. */
+			J9VMThread *currentThread = vm->internalVMFunctions->currentVMThread(vm);
```

Comment:
There's a `getCurrentThread` jvmti helper method to handle unattached threads correctly.  It should probably be used here as well.

Reply:
Agreed, use the helper method `getCurrentVMThread` here instead. 
For another usage of `currentThread` at `jvmtiHelpers.c:disposeEnvironment()` without a return error code, check if the `currentThread` is `NULL` directly, and skip the use of `currentThread` when it is unattached and has a `NULL` value.